audio/espeakedit/files/patch-espeakedit-mac.diff

```diff
@@ -1,30 +1,32 @@
 diff -ruEN orig/Makefile src/Makefile
 --- orig/Makefile	2020-06-25 21:54:53.000000000 +0700
-+++ src/Makefile	2020-06-26 09:34:19.000000000 +0700
-@@ -10,17 +10,18 @@
++++ src/Makefile	2020-06-29 11:28:35.000000000 +0700
+@@ -10,17 +10,16 @@
  
  OBJS=$(patsubst %.cpp,%.o,$(SRCS))
  
 -WX_LIBS = -pthread   `wx-config --libs`
-+WX_LIBS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --libs`
-+
-+WX_CFLAGS=`/opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config --cflags`
++WX_LIBS=`wx-config --libs`
```

Comment:
wxgtk-2.8 doesn't install /opt/local/bin/wx-config; it installs /opt/local/Library/Frameworks/wxWidgets.framework/Versions/wxGTK/2.8/bin/wx-config, so this doesn't work. From [the build log](https://paste.z0k.xyz/42467d52aad7):

```
/usr/bin/clang++ -O2 -Wall -fexceptions `wx-config --cflags` -DGTK_NO_CHECK_CASTS -D__WXGTK__ -D_FILE_OFFSET_BITS=64 -D_LARGE_FILES -D_LARGEFILE_SOURCE=1 -DNO_GCC_PRAGMA -D_ESPEAKEDIT -I. -c -fno-exceptions espeak_command.cpp
/bin/sh: wx-config: command not found
compiledata.cpp:26:10: fatal error: 'wx/wx.h' file not found
#include "wx/wx.h"
         ^~~~~~~~~
```

Reply:
So you either need to patch, as you did before, but leave a `@FRAMEWORKS_DIR@` placeholder that you can `reinplace` with `${frameworks_dir}` in a `post-patch` block, or you could add that directory to the path, e.g.:
```
configure.env-append PATH=${frameworks_dir}/wxWidgets.framework/Versions/wxGTK/2.8/bin:$env(PATH)
```
(Might possibly be needed in build and/or destroot phases as well; haven't tried it.)

Reply:
Using PortGroup wxWidgets-1.0 to resolve this.